### PR TITLE
Horizon/feature/19859 be move entity framework plus into nu get package

### PR DESCRIPTION
**Pull request by im-ashaban**
[AB#19859](https://dev.azure.com/Interfirst-DevOps/ec67a686-9dcc-441d-a2a0-3cc4609082d2/_workitems/edit/19859) [AB#19110](https://dev.azure.com/Interfirst-DevOps/ec67a686-9dcc-441d-a2a0-3cc4609082d2/_workitems/edit/19110)

### CODE REVIEW

File: src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj [hunk 1]
```diff
@@ -13,11 +13,13 @@
     <RootNamespace>Z.EntityFramework.Plus.EFCore.NETStandard21</RootNamespace>
     <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>zzzproject.pfx</AssemblyOriginatorKeyFile>
-    <PackageId>Z.EntityFramework.Plus.EFCore</PackageId>
+    <PackageId>Interfirt.Z.EntityFramework.Plus.EFCore</PackageId>
```
Comment (im-pmakul) [inline src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj]: Interfirt.Skynet.

File: src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj [hunk 2]
```diff
@@ -13,11 +13,13 @@
...
     <Version>3.0.40</Version>
```
Comment (im-pmakul) [inline src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj]: use please version according to current sprint

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# ASP.NET Core (.NET Framework)
```
Comment (im-pmakul) [inline azure-pipelines.yml]: use please pipeline such as LoanPipelines nuget pipeline

**Review by im-pmakul — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 3 above)
Comment (im-ogonchar) [reply azure-pipelines.yml]: [Here is the example](https://github.com/Interfirst/Interfirst.Skynet.LoanPipelines/blob/master/azure-pipelines-nuget.yml)

File: src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj (hunk 2 above)
Comment (im-ogonchar) [reply src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj]: In this case I think we can add initial version like `<PackageReleaseNotes>Version 3.0.40</PackageReleaseNotes>`

Comment (im-ashaban) [reply src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj]: I disagree with version comment, that's external package and you need to know at what version it based on

**Review by im-ogonchar — APPROVED** [review]

File: src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj (hunk 2 above)
Comment (im-pmakul) [reply src/Z.EntityFramework.Plus.EFCore3x.NETStandard21/Z.EntityFramework.Plus.EFCore3x.NETStandard21.csproj]: This is our (IM) package. And version related to our inner versioning. If you want to notify about smth you can do this with PackageReleaseNotes or Description

**Review by im-pmakul — APPROVED** [review]

**Pull request merged**